### ISSUE TITLE
 problems when  dooms themes work with org-mode and neotree

### ISSUE THREAD

**Issue by ichirika** [hlissner/emacs-doom-themes#86]
1. org-mode "*" heading with linum-relative has a problem that a little part of "0" is covered by "*"
2. with doom-neotree, last char of the root dir's name doesnt display.
here is the pic
![doom-theme-issue](https://user-images.githubusercontent.com/16498050/28304386-ee30cfd4-6bc9-11e7-8be7-1115fdc4d366.png)

anyway,i love the doom-themes, thanks for your effort.

**Comment by YuSuu** [comment]
It's very strange,
if i use "neotree-dir" command to open folder,
the second problem doesnt appear

**Comment by hlissner** [comment]
I've added a fix for the second issue. Let me know if it still occurs.

The linum-relative issue is an odd one. I've just pushed a few updates to line number faces, could you update and see if it fixed your issue?

**Comment by YuSuu** [comment]
Yeah.Second issue is fixed. Good work!
The first one issue still exists.

**Comment by ryankask** [comment]
The ellipsis is not rendering for me. Sometimes it does, however, and it appears like the unfilled rectangle at the top right of your screenshot.

Do you know what this is or how I can have a normal ellipsis?

**Comment by xiaoxinghu** [comment]
I also have the ellipsis problem, I am using [emacs-mac](https://bitbucket.org/mituharu/emacs-mac/overview).

**Comment by xiaoxinghu** [comment]
@ryankask In case you are looking for a temporary solution (it drives me nuts looking like that), set the `org-ellipsis` to `nil`.

**Comment by ryankask** [comment]
Thanks, @xiaoxinghu. I [made this change](https://github.com/ryankask/dotfiles/blob/master/.emacs.d/elisp/init-ui.el#L78) last week because aside from the strange appearance, it felt really slow when editing the file. emacs actually crashed once when opening an org file. 

I'm using macOS but I'm not sure what's going on.

**Comment by partnano** [comment]
I am also encountering the ellipsis problem, running Emacs 25.2 on Manjaro Linux
The suggested fix (setting the variable to nil) works, but it would be amazing if we could use a custom ellipsis with the theme

**Comment by hlissner** [comment]
doom-themes only changed the default and shouldn't stop you from changing it with `(setq org-ellipsis " ... ")`. Perhaps some of you are changing it _before_ the theme is loaded, which gets overwritten when the theme is later loaded?

In any case, I've removed this from doom-themes. Let me know if that resolves your issues.

**Comment by hlissner** [comment]
@ichirika Sorry for the long wait. You can fix problem #1 by either: a) disabling the differently sized org headers, like so:

```emacs-lisp
(set-face-attribute 'org-level-1 nil :height 1.0)
```

Or by explicitly setting an absolute `:height` for linum and linum-relative's faces:

```emacs-lisp
(let ((height (face-attribute 'default :height)))
  (set-face-attribute 'linum nil :height height)
  (set-face-attribute 'linum-relative-current-face nil :height height))
```

Unfortunately, this is a natural limitation of overlays, which linum (and nlinum) use to display line numbers. Issues like these should disappear once we get to Emacs 26, however, which has line numbers built in.

Problem #2 should have been resolved since https://github.com/hlissner/emacs-doom-themes/commit/fb92c7c31b490c5fce997024b9c9b8c05f607301, so I'll close this issue. Feel free to reopen if my suggestions above didn't help.
